manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -83,6 +83,12 @@ spec:
       regex: network_plugin_operations_latency_microseconds|sync_proxy_rules_latency_microseconds|rest_client_request_latency_seconds
       sourceLabels:
       - __name__
+    - action: replace
```

Comment:
This needs to be just a relabeling rule ("relabelings"), not a metric relabeling rule, then you also don't need to match on the metric label name at all, every metric will have the label.

Reply:
@brancz changed it to use `relabelings`.